src/compat/db_json.cpp

```diff
@@ -473,7 +489,7 @@ db_json_extract_document_from_path (JSON_DOC *document, const char *raw_path, JS
       return ER_JSON_INVALID_PATH;
     }
 
-  resulting_json = p.Get (*document);
+  resulting_json = const_cast<JSON_VALUE *> (p.Get (*document));
```

Comment:
Is the const_cast necessary?

Reply:
Because I changed the parameter **JSON_DOC*** to **const JSON_DOC*** the conversion is necessary. Otherwise there will be this error: cannot convert from 'const JSON_VALUE *' to 'JSON_VALUE *'